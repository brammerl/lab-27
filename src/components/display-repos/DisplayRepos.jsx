import React from 'react';

const DisplayRepos = ({userRepos}) => {
  userRepos.forEach(repository => {return (
    <>
      <h2>Repositories</h2>
      <ul>
        <li><a href={repository.html_url}>{repository.name}</a></li>
      </ul>
    </>
  )
})
};


export default DisplayRepos;