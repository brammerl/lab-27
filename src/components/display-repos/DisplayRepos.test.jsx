import React from 'react';
import { shallow } from 'enzyme';
import DisplayRepos from './DisplayRepos';

describe('DisplayRepos Component', () => {
  it('renders the display repo component', () => {
    const wrapper = shallow(<DisplayRepos userRepos={[]}/>)
    expect(wrapper).toMatchSnapshot();
  })
})