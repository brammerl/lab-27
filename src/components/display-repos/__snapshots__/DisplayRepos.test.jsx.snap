// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DisplayRepos Component renders the display repo component 1`] = `
<Fragment>
  <h2>
    Repositories
  </h2>
  <ul />
</Fragment>
`;
